src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToClaimSpecCallbackHandler.java

```diff
@@ -96,6 +97,20 @@ private CallbackResponse handleDefendAllClaim(CallbackParams callbackParams) {
         }
         if ("DEFENDANT_RESPONSE_SPEC".equals(callbackParams.getRequest().getEventId())) {
 
+            if ("HAS_PAID_THE_AMOUNT_CLAIMED".equals(caseData.getDefenceRouteRequired())
```

Comment:
I thought you had created a Constants class for this

Reply:
this was covered in Feature/roc 9698